test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -373,3 +377,66 @@ bool TestFunctionCall::matchesExpectation() const
 {
 	return m_failure == m_call.expectations.failure && m_rawBytes == m_call.expectations.rawBytes();
 }
+
+bool TestFunctionCall::checkFixedPointTypes()
```

Comment:
What does this do and why is it needed?

Reply:
Can you answer this?

Reply:
This function is used to check whether the supplied parameters of a test function call are correct with respect to the used parameter type. It also checks the return type. Right now it is only checking for the correct usage of fixed point type. Mismatches will result in an unmatching assertion, where also the corresponding error message will be set. Maybe we should use a different function name here.

Reply:
Why is it needed for fixed point types and not for anything else?